docs/changelog/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert/): expose `vmalert_alerts_send_duration_seconds` metric to measure the time taken to send alerts to the specified `-notifier.url`. Thanks to @eyazici90 for [the pull reuqest](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8468).
 
 * BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation): fix panic on `rate` output. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8469).
+* BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation): fix updating min timestamp threshold when deduplication and windows are enabled.
```

Comment:
As a user, even familiar with stream aggregation, I do not understand this changelog line. Would you mind explaining it in a simple words? Or by referring to the negative behavior that it fixes?

Reply:
with deduplication and windows enabled minTimestamp was updated twice in a loop, first time inside dedupFlush function to a valid value and second time in flush to a timestamp in a future. first flush has no issues as initial minTimestamp is valid, but subsequent flushes are empty